pymongo/__init__.py

```diff
@@ -87,11 +88,13 @@
 
 from pymongo import _csot
 from pymongo._version import __version__, get_version_string, version_tuple
-from pymongo.collection import ReturnDocument
-from pymongo.common import MAX_SUPPORTED_WIRE_VERSION, MIN_SUPPORTED_WIRE_VERSION
-from pymongo.cursor import CursorType
-from pymongo.mongo_client import MongoClient
-from pymongo.operations import (
+from pymongo.cursor_shared import CursorType
```

Comment:
CursorType should still be imported from `pymongo.cursor`.

Reply:
Good catch, fixed.